packages/connectivity/src/scp-cf/destination/destination-from-vcap.ts

```diff
@@ -53,28 +53,31 @@ export async function getDestinationFromServiceBinding(
     : undefined;
 
   const retrievalOptions = { ...options, jwt: decodedJwt };
-  const destination =
-    (await retrieveDestinationFromCache(retrievalOptions)) ||
-    (await retrieveDestinationWithoutCache(retrievalOptions));
+  let destination = await retrieveDestinationFromCache(retrievalOptions);
```

Comment:
I wonder what I was thinking when writing this code. Seems super-weird.
~[req]~ I think that this is not enough though, if we have two requests with the same destination and the first is called with `useCache = true`, the second with `useCache = false`, and the token is not yet expired it will use the destination from the cache, which is not supposed to do, right?

I guess an additional test for this would be helpful. 

*EDIT*: I just saw that `retrieveDestinationCache()` checks for `useCache`. This is not obvious and I think it might be confusing at this point in the code (again: why did I build it this way? 🤪)
[nth] I would prefer to drag the `useCache` check up to here - maybe we just don't need the `retrieveDestinationWithoutCache()` function.